syntax = "proto3";
package iroha.ordering.proto;

import "block.proto";
import "google/protobuf/empty.proto";

message Proposal {
  repeated iroha.protocol.Transaction transactions = 1;
}

service OrderingGate {
  rpc SendProposal (Proposal) returns (google.protobuf.Empty);
}

service OrderingService {
  rpc SendTransaction (iroha.protocol.Transaction) returns (google.protobuf.Empty);
}